json_serializable/lib/src/type_helpers/map_helper.dart

```diff
@@ -56,9 +59,9 @@ class MapHelper extends TypeHelper<TypeHelperContextWithConfig> {
     _checkSafeKeyType(expression, keyArg);
 
     final valueArgIsAny = _isObjectOrDynamic(valueArg);
-    final isEnumKey = isEnum(keyArg);
+    final isKeyStringable = _isStringKeyable(keyArg);
```

Comment:
why are the words flipped? What does Stringable mean? What does Keyable mean?

Reply:
Fix the function name. Added docs there.